File: tests/Contexts/Rutine/Exercises/domain/ExerciseMuscleMother.ts
import { ExerciseMuscle } from '../../../../../src/Contexts/Rutine/Exercises/domain/ExerciseMuscle';
import { WordMother } from '../../../Shared/domain/WordMother';

export class ExerciseMuscleMother {
  static create(value: string): ExerciseMuscle {
    return new ExerciseMuscle(value);
  }

  static random(): ExerciseMuscle {
    return this.create(WordMother.random());
  }
}

File: src/test/java/com/kuro/rutineweb/exercises/entities/ExerciseMuscleMother.java
package com.kuro.rutineweb.exercises.entities;

import com.kuro.rutineweb.Shared.entities.WordMother;

public class ExerciseMuscleMother {
    public static ExerciseMuscle create(String value) {
        return new ExerciseMuscle(value);
    }

    public static ExerciseMuscle random() {
        return new ExerciseMuscle(WordMother.random(30));
    }
}
